extensions/wikia/DesignSystem/scripts/DesignSystemAuthenticationMenu.js

```diff
@@ -0,0 +1,36 @@
+$(function ($) {
+	'use strict';
+
+	var $signInButton = $('#global-navigation-anon-sign-in'),
+		$registerButton = $('#global-navigation-anon-register');
+
+	function onAuthSuccess() {
+		var redirect = this.url.replace(/.*?redirect=([^&]+).*/, '$1');
+		window.location.href = decodeURIComponent(redirect);
+	}
+
+	function openAuthModal(event) {
+		var url = event.currentTarget.href;
+
+		//Prevent opening modal with shift / alt / ctrl / let only left mouse click
+		if (event.which !== 1 || event.shiftKey || event.altKey || event.metaKey || event.ctrlKey) {
```

Comment:
Yes it was copied from old global navigation i could paste the link although source would be deleted soon